### PR TITLE
[word] (Requirement Sets) Splitting Word requirement set pages

### PR DESCRIPTION
**Pull request by AlexJerabek**
This matches the new look of the Excel requirement set pages.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0ed6292](https://github.com/OfficeDev/office-js-docs-pr/pull/1185/commits/0ed62929e29cfaa8d5987127485e04b24c019e44)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/openspec/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/openspec/openspec.md) | :warning:Warning | | [Details](#user-content-9b520580-ec9ccc54)
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-1-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-1-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-2-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-2-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-3-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-3-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-preview-apis.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="9b520580-ec9ccc54"></a>
### [docs/reference/openspec/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/openspec/openspec.md)
 - **Line 26**: **[Warning]** `````Illegal link: `<a href="reference/requirement-sets/word-api-requirement-sets.md#word-javascript-preview-apis">Word JavaScript API preview requirement set</a>` -- missing bookmark. The file reference/requirement-sets/word-api-requirement-sets.md doesn't contain a bookmark named word-javascript-preview-apis.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172355051568-1185%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yhz%2B3Wa%2BDYyQwcU%2B9YGsKKkOVrHX%2FZEYNh3gKHZYKBc%3D&st=2019-07-17T23%3A51%3A13Z&se=2019-08-17T23%3A56%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f9e89c7](https://github.com/OfficeDev/office-js-docs-pr/pull/1185/commits/f9e89c76ddc757cb1fad895d78ef44a31343f506)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/openspec/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/openspec/openspec.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-1-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-1-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-2-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-2-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-3-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-3-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-preview-apis.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907180005026713-1185%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kKDUSQwMPyveF5vOcSxpvk8QORiUvWz3TpZDpbOB588%3D&st=2019-07-18T00%3A01%3A19Z&se=2019-08-18T00%3A06%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/word-api-requirement-sets.md [hunk 1]
```diff
@@ -10,19 +10,21 @@ localization_priority: Priority
...
-| WordApi 1.1  | Version 1509 (Build 4266.1001) or later| January 2016, 1.18 or later | January 2016, 15.19 or later| September 2016 | N/A |
+|  Requirement set  |   Office on Windows\*<br>(connected to Office 365 subscription)  |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  |
+|:-----|-----|:-----|:-----|:-----|
+| [Preview](word-preview-apis.md) | Please use the latest Office version to try preview APIs (you may need to join the [Office Insider program](https://products.office.com/office-insider)) |
+| [WordApi 1.3](word-api-1-3-requirement-set.md) | Version 1612 (Build 7668.1000) or later| March 2017, 2.22 or later | March 2017, 15.32 or later| March 2017 |
+| [WordApi 1.2](word-api-1-2-requirement-set.md) | December 2015 update, Version 1601 (Build 6568.1000) or later | January 2016, 1.18 or later | January 2016, 15.19 or later| September 2016 |
+| [WordApi 1.1](word-api-1-1-requirement-set.md) | Version 1509 (Build 4266.1001) or later| January 2016, 1.18 or later | January 2016, 15.19 or later| September 2016 |
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-requirement-sets.md]: Suggest that you add the **Office versions and build numbers** H2 heading after the note that appears below this table, and before the sentence "To find out more about versions,...." -- like the Excel requirement sets overview article does:

image

Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-requirement-sets.md]: Also suggest you add a link at the top of the **See also** section for `[Word JavaScript API reference documentation](/javascript/api/word)`

File: docs/reference/requirement-sets/word-api-1-3-requirement-set.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+# What's new in Word JavaScript API 1.3
+
+WordApi added more support for content controls, custom XML, and document-level settings.
+
+## API List
+
+The following is a complete list of APIs added as part of the WordApi 1.3 requirement set.
+
+| Class | Fields | Description |
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-3-requirement-set.md]: There's a problem with the formatting of this table that's resulting in it not rendering correctly.  

https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/word-api-1-3-requirement-set?branch=AlexJ-WorReqSets

image

File: docs/reference/requirement-sets/word-api-1-3-requirement-set.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+title: Word JavaScript API requirement set 1.3
+description: 'Details about the WordApi 1.3 requirement set'
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.3
+
+WordApi added more support for content controls, custom XML, and document-level settings.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-3-requirement-set.md]:
```suggestion
WordApi 1.3 added more support for content controls, custom XML, and document-level settings.
```

File: docs/reference/requirement-sets/word-api-1-3-requirement-set.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+
+## API List
+
+The following is a complete list of APIs added as part of the WordApi 1.3 requirement set.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-3-requirement-set.md]:
```suggestion
The following tables lists the APIs added as part of the WordApi 1.3 requirement set.
```

File: docs/reference/requirement-sets/word-api-1-2-requirement-set.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+title: Word JavaScript API requirement set 1.2
+description: 'Details about the WordApi 1.2 requirement set'
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.2
+
+WordApi added support for inline pictures.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-2-requirement-set.md]:
```suggestion
WordApi 1.2 added support for inline pictures.
```

File: docs/reference/requirement-sets/word-api-1-2-requirement-set.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## API List
+
+The following is a complete list of APIs added as part of the WordApi 1.2 requirement set.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-2-requirement-set.md]:
```suggestion
The following table lists the APIs added as part of the WordApi 1.2 requirement set.
```

File: docs/reference/requirement-sets/word-api-1-3-requirement-set.md [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.3
+
+WordApi added more support for content controls, custom XML, and document-level settings.
+
+## API List
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-3-requirement-set.md]:
```suggestion
## API list
```

File: docs/reference/requirement-sets/word-api-1-2-requirement-set.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.2
+
+WordApi added support for inline pictures.
+
+## API List
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-2-requirement-set.md]:
```suggestion
## API list
```

File: docs/reference/requirement-sets/word-api-1-1-requirement-set.md [hunk 9]
```diff
@@ -0,0 +1,226 @@
...
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.1
+
+Word JavaScript API 1.1 is the first version of the API. It is the only Word-specific requirement set supported by Excel 2016.
+
+## API List
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-1-requirement-set.md]:
```suggestion
## API list
```

File: docs/reference/requirement-sets/word-api-1-1-requirement-set.md [hunk 10]
```diff
@@ -0,0 +1,226 @@
...
+title: Word JavaScript API requirement set 1.1
+description: 'Details about the WordApi 1.1 requirement set'
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.1
+
+Word JavaScript API 1.1 is the first version of the API. It is the only Word-specific requirement set supported by Excel 2016.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-1-requirement-set.md]: In the other requirement set articles, you refer to the set using this convention: "WordApi 1.3".  For consistency, should "Word JavaScript API 1.1." in this sentence be changed to -> "WordApi 1.1" ?  The suggestion I've made below incorporates this change and a couple of other wording changes as well (including "Excel" -> "Word").

Comment (kbrandl) [reply docs/reference/requirement-sets/word-api-1-1-requirement-set.md]:
```suggestion
WordApi 1.1 is the first requirement set of the Word JavaScript API. It's the only Word API requirement set supported by Word 2016.
```

File: docs/reference/requirement-sets/word-api-1-1-requirement-set.md [hunk 11]
```diff
@@ -0,0 +1,226 @@
...
+
+## API List
+
+The following is a complete list of APIs added as part of the WordApi 1.1 requirement set.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-api-1-1-requirement-set.md]:
```suggestion
The following table lists the APIs added as part of the WordApi 1.1 requirement set.
```

File: docs/reference/requirement-sets/word-preview-apis.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+# Word JavaScript preview APIs
+
+New Word JavaScript APIs are first introduced in "preview" and later become part of a specific, numbered requirement set after sufficient testing occurs and user feedback is acquired.
+
+> [!NOTE]
+> Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
+>
+> To use preview APIs, you must reference the **beta** library on the CDN (https://appsforoffice.microsoft.com/lib/beta/hosted/office.js) and you may also need to join the Office Insider program to get a recent Office build.
+
+## API List
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-preview-apis.md]:
```suggestion
## API list
```

File: docs/reference/requirement-sets/word-preview-apis.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+
+The following is a complete list of APIs currently in preview.
```
Comment (kbrandl) [inline docs/reference/requirement-sets/word-preview-apis.md]:
```suggestion
The following table lists the APIs that are currently in preview.
```

File: docs/reference/requirement-sets/excel-api-requirement-sets.md [hunk 14]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Excel JavaScript API requirement sets
 description: 'Office Add-in requirement set information for Excel builds'
-ms.date: 07/11/2019
+ms.date: 07/17/2019
```
Comment (kbrandl) [inline docs/reference/requirement-sets/excel-api-requirement-sets.md]: Suggest that you not update ms.date in this article, as the changes have no substantive affect on the article's content; we don't want to give readers the impression that something has changed in the article when, in fact, nothing has really changed.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[60bd156](https://github.com/OfficeDev/office-js-docs-pr/pull/1185/commits/60bd156f4c5154f8b8b74e05599abcb1e2995107)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/openspec/openspec.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/openspec/openspec.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-1-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-1-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-2-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-2-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-1-3-requirement-set.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-1-3-requirement-set.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-api-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/word-preview-apis.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/reference/requirement-sets/word-preview-apis.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-WorReqSets/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907181830467984-1185%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=X5Pa5UsmWGiOEUKfeX7bATbWqxR%2BCFs8i0Tuf867ZVQ%3D&st=2019-07-18T18%3A26%3A59Z&se=2019-08-18T18%3A31%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @kbrandl. It's curious how the tables broke on the staging site and not in the VS Code Markdown preview.

**Pull request merged**